src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/metrics/CollectQualityYieldMetricsSparkIntegrationTest.java

```diff
@@ -22,21 +23,38 @@
     @DataProvider(name = "CollectQualityYieldMetrics")
     private Iterator<Object[]> makeCollectQualityYieldMetricsData(){
         final List<Object[]> list= new ArrayList<>();
-        list.add(new Object[]{"collect_quality_yield_metrics.sam", "collect_quality_yield_metrics.originalquals.txt", true});
-        list.add(new Object[]{"collect_quality_yield_metrics.sam", "collect_quality_yield_metrics.quals.txt", false});
+
+        list.add(new Object[]{"valid.bam", "valid.CollectQualityYieldMetrics.txt", null, true});
+        list.add(new Object[]{"valid.cram", "valid.CollectQualityYieldMetrics.txt", new File(TEST_DATA_DIR, "valid.fasta").getAbsolutePath(), true});
+
+        list.add(new Object[]{"collect_quality_yield_metrics.sam", "collect_quality_yield_metrics.originalquals.txt", null, true});
+        list.add(new Object[]{"collect_quality_yield_metrics.sam", "collect_quality_yield_metrics.quals.txt", null, false});
+
+        //Those two work on non-spark version https://github.com/broadinstitute/gatk/issues/1262
```

Comment:
This ticket was closed as invalid, since it was too broad -- can you open a new ticket against just this metric?